notebooks/1_spikesorting.ipynb

```diff
@@ -4,26 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.                                       'lab_member_name' : your_name}, skip_duplicates=True)
</pre><p>If you're gonna use skip_duplicates=True, why don't you just make this cell shorter by just doing an insert (without all these ifs)? and then you can have the user check that it worked. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/nwb_datajoint/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>I'd prefer to not have the user have to check anything, but instead show them. I don't really think they should be doing the leg work in an intro tutorial, but happy to change if you feel differently!</p>